linux_os/guide/system/network/network-nftables/nftables_rules_permanent/ansible/sle15.yml

```diff
@@ -0,0 +1,19 @@
+# platform = sle15
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+{{% set nftables_family_names = ['bridge', 'arp', 'inet'] %}}
+{{{ ansible_instantiate_variables("var_nftable_master_config_file") }}}
+
+- name: Check the top-level configuration file exists
+  stat:
+    path: "{{ var_nftable_master_config_file }}"
+
+{{% for family in nftables_family_names %}}
+- name: Check the {{{ family }}} relevant file is included configuration
+  lineinfile:
```

Comment:
please use the full qualified name Ansible module name in new Ansible content.

Reply:
:+1: done in d7a048a